apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/viderefoere-opphoer/ViderefoereOpphoerVurdering.tsx

```diff
@@ -53,6 +57,7 @@ export const ViderefoereOpphoerVurdering = ({
   const [begrunnelse, setBegrunnelse] = useState<string>(viderefoertOpphoer?.begrunnelse || '')
   const [setViderefoertOpphoerStatus, setViderefoertOpphoer, resetToInitial] = useApiCall(lagreViderefoertOpphoer)
   const [kravdato] = useState<string | undefined>()
+  const [, slettViderefoertOpphoerCall] = useApiCall(slettViderefoertOpphoer)
```

Comment:
ha en visning for feil i endepunktet hvis det oppstår?

Reply:
Done!